data/adapters/file_fileset_adapter.py

```diff
@@ -0,0 +1,260 @@
+            if biosample:
+                sample_ids.add(biosample['accession'])
+                sample_term_id = biosample['biosample_ontology']['term_id']
+                if sample_term_ids and sample_term_id not in sample_term_ids:
+                    raise(ValueError(
+                        f'Biosample type of the dataset is not the same as the biosamples.'))
+                else:
+                    sample_term_ids.add(sample_term_id)
+                simple_sample_summary = biosample['biosample_ontology']['term_name']
+                donor = biosample.get('donor')
+                if donor:
+                    donor_accession = donor['accession']
+                    donor_ids.add(donor_accession)
+                    if biosample['biosample_ontology']['classification'] in ['in vitro differentiated cells', 'primary cell', 'tissue', 'organoid']:
+                        simple_sample_summary = f'{simple_sample_summary} from {donor_accession}'
+                biosample_treatments = biosample.get('treatments', [])
+                if biosample_treatments:
+                    treatment_term_names = set()
+                    for treatment in biosample_treatments:
+                        treatment_id = treatment.get('treatment_term_id')
+                        if treatment_id:
+                            treatment_ids.add(treatment_id)
+                        treatment_term_names.add(
+                            treatment['treatment_term_name'])
+                    treatment_term_names = ', '.join(
+                        sorted(list(treatment_term_names)))
+                    simple_sample_summary = f'{simple_sample_summary} treated with {treatment_term_names}'
+                simple_sample_summaries.add(simple_sample_summary)
+    if not(simple_sample_summaries) and biosample_type_term:
+        simple_sample_summary = f'{biosample_type_term}'
+        donor = dataset_object.get('donor', '')
+        if donor:
+            donor_object = requests.get(
+                portal_url + donor + '/@@object?format=json').json()
+            donor_accession = donor_object['accession']
+            donor_ids.add(donor_accession)
+            simple_sample_summary = f'{simple_sample_summary} from {donor_accession}'
+        treatments = dataset_object.get('treatments', [])
+        treatment_term_names = set()
+        for treatment in treatments:
+            treatment_id = treatment.get('treatment_term_id')
+            if treatment_id:
+                treatment_ids.add(treatment_id)
+                treatment_term_names.add(treatment['treatment_term_name'])
+        if treatment_term_names:
+            treatment_term_names = ', '.join(
+                sorted(list(treatment_term_names)))
+            simple_sample_summary = f'{simple_sample_summary} treated with {treatment_term_names}'
+        simple_sample_summaries.add(simple_sample_summary)
+
+
+def query_fileset_files_props_encode(accession):
```

Comment:
the new methods can be moved into the `FileFileSet` class, for example:

`def query_fileset_files_props_encode(self, accession):`

and the call would be: `self.query_fileset_files_props_encode(self.accession)` on line 29